change-notes/1.25/analysis-csharp.md

```diff
@@ -24,5 +24,28 @@ The following changes in version 1.25 affect C# analysis in all applications.
   have type parameters. This means that non-generic nested types inside construced types,
   such as `A<int>.B`, no longer are considered unbound generics. (Such nested types do,
   however, still have relevant `.getSourceDeclaration()`s, for example `A<>.B`.)
+* The data-flow library has been improved, which affects and improves most security queries. Flow
```

Comment:
~~affects and~~

I would also state exactly how it improves it, for example that it will find more data-flow steps and potentially more results.